packages/gatsby/src/redux/reducers/index.js

```diff
@@ -53,7 +53,7 @@ module.exports = {
   nodesByType: getNodesByTypeReducer(),
   resolvedNodesCache: require(`./resolved-nodes`),
   nodesTouched: require(`./nodes-touched`),
-  lastAction: require(`./last-action`),
+  lastAction: require(`./last-action`).lastAction,
```

Comment:
You can import that like e.g. `statusReducer`, `logReducer` etc.

Reply:
Done!
I had not made this change at this time because my next PR was to convert this entire file to TS.